.nycrc

```diff
@@ -10,7 +10,6 @@
     "**/samples",
     "**/scripts",
     "**/src/**/v*/**/*.js",
-    "**/protos",
```

Comment:
:( This brings down our test coverage from 96% to 60%ish. @bcoe - Can you point me at the source of this change so that I can update the configuration?